### PR TITLE
Configure PVS-proxy on start/stop

### PR DESCRIPTION
**Pull request by robhoes**

### CODE REVIEW

**Comment by robhoes** [comment]
Need xenserver/xcp-idl#5.

File: ocaml/xapi/xapi_pvs_proxy.ml [hunk 1]
```diff
@@ -15,37 +15,57 @@
...
+			Xapi_pvs_farm.update_farm_on_localhost ~__context ~self:farm ~vdi ~stopping_proxies:[proxy] ();
+			Network.Net.Bridge.remove_port dbg ~bridge ~name:port_name;
+			Db.PVS_proxy.set_currently_attached ~__context ~self:proxy ~value:false;
+			Db.PVS_proxy.set_cache_SR ~__context ~self:proxy ~value:Ref.null
+		with e ->
+			let reason =
+				match e with
+				| Xapi_pvs_cache.No_cache_vdi_present -> "no PVS cache VDI found"
+				| Network_interface.PVS_proxy_connection_error -> "unable to connect to PVS proxy daemon"
+				| _ -> "unknown error"
```
Comment (lindig) [inline ocaml/xapi/xapi_pvs_proxy.ml]: you could use `Printf.sprintf "unknown error (%s)" __LOC__` to include the location and also turn the exception into a string

Comment (robhoes) [inline ocaml/xapi/xapi_pvs_proxy.ml]: Indeed, I'll use `Printexc.to_string e`.

**Comment by robhoes** [comment]
@lindig @johnelse Could you review this please?

**Comment by johnelse** [comment]
I think this looks good.

**Pull request merged**